privacyidea/static/components/config/views/config.policies.details.html

```diff
@@ -403,6 +403,9 @@ <h3 translate>Action</h3>
                                             ng-bind-html="action.desc | highlight: action_filter">
                                             </p>
                                         </td>
+                                        <td>
+                                            <a href="{{ 'https://privacyidea.readthedocs.io/en/latest/' + selectedScope + '.html#' + action.name }}">help</a>
```

Comment:
We have these nice (i) icons.

    <span class="glyphicon glyphicon-info-sign"></span>

we can preceed the help link with that.

Reply:
i think this is already in master... will check.

Reply:
Done in 8821f3bd